src/components/NavTop/NavTop.js

```diff
@@ -21,8 +32,10 @@ class NavTop extends Component {
     this.setState({ activateMyPage: false });
   };
 
-  // }
   render() {
+    const { myPageMenu } = this.state;
+    const { activateMyPage } = this.state;
```

Comment:
`const { myPageMenu, activateMyPage } = this.state;`

Reply:
🥳